st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -33,18 +33,14 @@
 from st2reactor.sensor.base import Sensor, PollingSensor
 from st2reactor.sensor import config
 from st2common.services.datastore import DatastoreService
+from st2common.util.monkey_patch import monkey_patch
 
 __all__ = [
     'SensorWrapper',
     'SensorService'
 ]
 
-eventlet.monkey_patch(
-    os=True,
-    select=True,
-    socket=True,
-    thread=False if '--use-debugger' in sys.argv else True,
-    time=True)
+monkey_patch()
```

Comment:
Any objections to using this wrapper method everywhere else in the code where we call `eventlet.monkey_patch` right now?